Portal/vip-application/src/main/java/fr/insalyon/creatis/vip/application/client/view/common/AbstractSourceLayout.java

```diff
@@ -152,4 +152,21 @@ public boolean isOptional() {
     public abstract void setValue(String value);
     
     public abstract boolean validate();
+    
+    /**
+     * Activates all fields of an optional input, in a relaunched simulation context.
+     * Avoids implementation in subtypes which are not concerned.
+     * 
+     */
+    public void enableInputRelaunchedSimulation() {

```

Comment:
Do not find it useful to have "relaunchedSImulation" in the method name. "enableSource" or "enable" is better in my opinion.

Reply:
Ok. "enableSource" or "enableInput" seem to be good. In the last commit, I propose "enableInput" because method implementations are (or will) be in classes "Input..."